openjdk/src/test/java/org/conscrypt/MoreAssertions.java

```diff
@@ -0,0 +1,23 @@
+package org.conscrypt;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class MoreAssertions {
+  public static void assertContentsEqual(Collection<?> expected, Collection<?> actual) {
+    assertContentsEqual(null, expected, actual);
+  }
+
+  public static void assertContentsEqual(
+      String message, Collection<?> expected, Collection<?> actual) {
+    assertEquals(message, expected.size(), actual.size());
+    assertTrue(message, actual.containsAll(expected));
+    List<Object> elements = new ArrayList<>(actual);
+    elements.removeAll(expected);
+    assertEquals(message, 0, elements.size());
```

Comment:
Is this really worth adding a new test helper class for?

This doesn't seem to do anything different from

   assertEquals(message, new HashSet<>(expected), new HashSet<>(actual))

except that it has a worse error message in case of failure. No?